lightning/src/util/config.rs

```diff
@@ -427,7 +427,7 @@ impl Default for ChannelConfig {
 			forwarding_fee_proportional_millionths: 0,
 			forwarding_fee_base_msat: 1000,
 			cltv_expiry_delta: 6 * 12, // 6 blocks/hour * 12 hours
-			max_dust_htlc_exposure_msat: 5_000_000,
+			max_dust_htlc_exposure_msat: 25_000_000,
```

Comment:
Note there is `test_max_dust_htlc_exposure`, and few other places to update in consequence.

Reply:
Thanks